be/src/exec/pipeline/scan/jdbc_scan_operator.cpp

```diff
@@ -94,7 +94,7 @@ void JDBCScanOperator::_start_scanner(RuntimeState* state) {
     scan_ctx.passwd = jdbc_table->jdbc_passwd();
     scan_ctx.sql = get_jdbc_sql(jdbc_table->jdbc_table(), _jdbc_scan_node.columns, _jdbc_scan_node.filters, _limit);
 
-    _scanner.reset(new vectorized::JDBCScanner(scan_ctx, _result_tuple_desc, this));
+    _scanner.reset(new vectorized::JDBCScanner(scan_ctx, _result_tuple_desc, this->unique_metrics()));
```

Comment:
It seems that `JDBCScanOperator::_start_scanner` can be removed? after you change, we don't need such an async thread to fetch data